src/app/client/src/app/modules/resource/components/collection-player/collection-player.component.ts

```diff
@@ -130,7 +134,7 @@ export class CollectionPlayerComponent implements OnInit, OnDestroy {
         object: {
           id: content.metadata.identifier,
           type: content.metadata.contentType || content.metadata.resourceType || content,
-          ver: content.metadata.pkgVersion ? content.metadata.pkgVersion.toString() : '1.0',
+          ver: content.metadata.pkgVersion || '1',
```

Comment:
why is this changed? previously, we were using "1.0" as the default version number.

Reply:
Merge changes